recipes/_configure_server.rb

```diff
@@ -55,8 +58,10 @@
     server_stanzas[stanza] = {}
     server_stanzas[stanza]['master_uri'] = master_uri
     server_stanzas[stanza]['pass4SymmKey'] = CernerSplunk::ConfTemplate.compose encrypt_password, CernerSplunk::ConfTemplate::Value.constant(value: pass) unless pass.empty?
-    # Until we support multisite clusters, set multisite explicitly false
-    server_stanzas[stanza]['multisite'] = false
+    if CernerSplunk.multisite_cluster?(bag, cluster)
+      server_stanzas[stanza]['multisite'] = true
+      server_stanzas[stanza]['site'] = bag['disable_search_affinity'] == true ? 'site0' : bag['site']
+    end
```

Comment:
ELSE, if it's not a multisite_cluster, I think we should still explicitly set multisite=false this was to get rid of a very noisy warning in the _internal logs if I remember the impetus behind 16e3b9949ba8a3100600166b20769bb5572d50c1 correctly

Reply:
Yep, I see it on the SH.
<img width="1399" alt="screen shot 2017-11-30 at 12 16 41 pm" src="https://user-images.githubusercontent.com/6389596/33447374-b522fd7a-d5c8-11e7-9a78-409eecdfd234.png">

Reply:
037658749c0b3eff411f800c343d2416e75145a3